lib/views/help/contact.html.erb

```diff
@@ -89,6 +89,35 @@
     </div>
 
 
+    <h2 class="contact-page__goal" id="wdtk">
+        <label class="houdini-label" for="goal3">
+            I’d like to get involved and become a <strong>volunteer</strong>
+        </label>
+    </h2>
+
+    <input class="houdini-input" type="radio" name="goals" id="goal3"
+        <% if params["contact"] && params[:current_form] == 'wdtk' %>checked<% end %>>
+    <div class="houdini-target contact-page__options">
+        <ul>
+            <li>
+                Please use this form if you are interested in volunteering for
+                WhatDoTheyKnow
+            </li>
+            <li>
+                If you would like more information on what volunteering
+                involves, see
+                <%= link_to _("our get involved page"), help_volunteers_path %>.
+            </li>
+            <li>
```

Comment:
Could put the form in a div rather than li to avoid the issue with the bullet, but you may have already tried that. No big deal.

Reply:
Going to leave as is. This is consistent with the other elements on the page.